src/plugins/home/public/application/application.tsx

```diff
@@ -30,15 +30,27 @@ export const renderApp = async (
 ) => {
   const homeTitle = i18n.translate('home.breadcrumbs.homeTitle', { defaultMessage: 'Home' });
   const { featureCatalogue, chrome } = getServices();
-  const navLinks = chrome.navLinks.getAll();
```

Comment:
we have this same logic in `kibana_overview` plugin as well (that's the Analytics home page): https://github.com/elastic/kibana/blob/main/src/plugins/kibana_overview/public/application.tsx#L27

I assume the same issue would occur there then? Can you add a fix for it there as well?